visualization/app/codeCharta/util/gameObjectsParser/gameObjectsImporter.ts

```diff
@@ -118,20 +152,19 @@ function createAttributeTypes() {
 	} as AttributeTypes
 }
 
-function createBaseGameObjectPosition(gameObjectPositions) {
-	const rootGamePosition = gameObjectPositions.find(gameObject => gameObject.name === "root")
-	const longEdge = Math.max(rootGamePosition.scale.x, rootGamePosition.scale.z)
+function createBaseGameObjectPosition(rootGameObjectPosition) {
+	const longEdge = Math.max(rootGameObjectPosition.scale.x, rootGameObjectPosition.scale.z)
 	return {
-		name: "base",
+		name: BASE_NAME,
 		position: {
 			x: 0,
 			y: 0,
 			z: 0
 		},
 		scale: {
 			x: longEdge,
-			y: 1,
+			y: 0,
```

Comment:
Was that your intention?

Reply:
We don't need a height. It's a folder and folders don't need heights. In addition, it has no effect if there is a 0, 50, 100, ...